public/hackathon/index.html

```diff
@@ -131,6 +136,8 @@ <h2>Výzvy</h2>
                                     Správa informačních technologií města Plzně
                                 </div>
                                 <div class="challenge-description">
+                                    Správa informačních technologií města Plzně momentálně připravuje novou verzi Životních situací - systéu navazujícímu například na uradbezcekani.cz, který by měl občanům Plzně usnadnit vyřizování věcí na úřadech. První část tohoto projektu je webový rozcestník - místo, kde si člověk podle situace, ve které se nachází (například se chce rozvést) může najít, co všechno by měl udělat a s úřady vyřešit.
+                                    Druhou částí (která je předmětem tohoto hackthonového téma) je i vyřešení dané situace online. Člověk místo obíhání úřadů bude moci rovnou vyplnit online formulář a například skrze datovou schránku ho předat na úřad, kde už za něj vše vyřeší.
```

Comment:
**Problém 1:** 
Tady to vypadá, jako by zde měl být nový odstavec, ale chybí tam `<p>` nebo `<br>`, takže se to ve výsledné stránce neprojeví.
Je to i u ostatních přidaných textů, kde to ale vypadalo jen jako zalomení zdrojového kódu. Zde jsem si ale už celkem jist, že je zde úmysl vytvořit zalomení ve stránce. 
Nevím tedy, zda je to bug, nebo záměr.

**Problém 2:**
Věta v závorce nese překlep a celkově nedává smysl: `(která je předmětem tohoto hackthonového téma)`.